docs/docs/userguide/CONFIGURATION.md

```diff
@@ -61,6 +61,9 @@ volumes:
 
 As explained above, this default configuration is suitable only for evaluation or testing purposes. Proper volume management needs to be configured.
 
+!!!info "Bitbucket default configuration exception"
+    In case of Bitbucket, `emptyDir` will not be created as a `shared-home` default volume. This is to utilise a Bitbucket's feature which allows `local-home`'s subpath (`<LOCAL_HOME_DIRECTORY>/shared`) to be `share-home` directory when `share-home` volume is not provided. Note that this is for single bitbucket node user only and using persistence volume for `local-home` is highly recommended for a production environment.
```

Comment:
I think this could be a bit more succinct, maybe something like:

```suggestion
For single node Bitbucket deployments, if no `shared-home` volume is defined, then a subpath of `local-home` will automatically be used for this purpose, namely: `<LOCAL_HOME_DIRECTORY>/shared`. This is behaviour specific to Bitbucket itself and is not orchestrated via the Helm chart.
```

Reply:
Thanks @jjeongatl , I see you have documented the use case is only for single node. Technically, we can enforce replica count to 1 if user set it to more than one as well as an action. However, I think this is not a high priority as we have the documentation and leave it to to decide.